### PR TITLE
Panguin fix

### PR DESCRIPTION
**Pull request by leafybillow**
A branch created to fix title problem for panguin

### CODE REVIEW

**Comment by leafybillow** [comment]
> I think one or two of the deleted .cfg and .C files can stay actually (see the few commits I made today on operations branch).

Of course, those files can stay in operations, or be kept in feature_panguin branches for further testing.

> What is an MD5?

https://en.wikipedia.org/wiki/MD5 . A histogram needs a unique name to avoid ambiguity when FindObject is called. Converting customized title into an MD5 seems to be the easiest way ( and may not be the best)  to fix this problem.

**Comment by cipriangal** [comment]
This looks like a needed improvement to me. Thanks @leafybillow !

**Comment by paulmking** [comment]
The one other thing which would be useful would be to go through the removed files from the "cleanup" commit, and add back the ones which are relevant.  @cameronc137 , you mentioned making some changes on operations in these files, would you please check that the files you care about haven't been removed here or restore them if they have been?
Or we could just revert the "cleanup" commit and do another cleanup in discussion with all parties.

**Comment by cameronc137** [comment]
@paulmking can we not include Tao's cleanup commit and instead merge the 3 commits I placed in Operations (9d99837a136d9345464495bacc8307b18a1e4e9d, 5d68a227a0f51e6521df39c39487d0c2113c54af, and bb1294a28ddc83ef7e788b06005988621e1a7908) as the updates to the Panguin Macros folder? This allows for preserving the wanted macros as well as adding the new Prminput files into develop. This can be done in a separate pull request.

**Comment by cipriangal** [comment]
@leafybillow: My changes from the other pull request created the conflict. Your issue might be fixed with what I have but please try to take a look to confirm.

@cameronc137 : if you want some changes I would suggest you either make changes to this pull request or just create a different branch and make your own pull request. We'll just have to be careful about conflict resolution.

**Comment by cameronc137** [comment]
Brought panguin_fix up to speed with operations.

**Comment by leafybillow** [comment]
@cameronc137 
Source file conflicts from this pull request and you should open a new pull request to merge macros update.

**Comment by cameronc137** [comment]
@cipriangal Can you look at the panguinOnline.cc conflict? I'm not sure which extra lines are duplicating efforts in there.

I cherry-picked the commits I added, so that they shouldn't conflict when merging operations in. I had already committed them to operations, so a pull-request onto here would accomplish the same thing I hope.

**Comment by cipriangal** [comment]
Ok ... @leafybillow : I need you to check this. I left the workaround for the ">>histName" option you had. 

But I removed the fixed draw options you set with your if statements. Not clear why the "scat" option was not working (maybe it needs to be capitalized?! https://root.cern/doc/v610/classTHistPainter.html#HP01c). Either way I think we should encourage everyone to not use any options if they want a scatter plot and use the appropriate drawing option (that is why the root documentation is there). Forcing everyone to do boxes if they want a 2D plot circumvents having the ability to set an option. 

If you are ok with this we can go ahead and merge this into develop.

File: panguin/macros/bcm_calibrated.cfg [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    title BCM vs Time Downstream
+    yield_bcm_an_ds:Entry$ ErrorFlag==0 -type . -tree mul -title "BCM AN Downstream vs Entry Number"
+    yield_bcm_dg_ds:Entry$ ErrorFlag==0 -type . -tree mul -title "BCM DG Downstream vs Entry Number"
+    yield_bcm_an_ds3:Entry$ ErrorFlag==0 -type . -tree mul -title "BCM3 Downstream vs Entry Number"
+    yield_bcm_an_ds10:Entry$ ErrorFlag==0 -type . -tree mul -title "BCM10 Downstream vs Entry Number"
+
+newpage 1 2
+    title BCM Upstream:Downstream Correlations
+    #yield_bcm_an_us:yield_bcm_an_ds ErrorFlag==0 -type . -tree mul -title BCM AN: "Upstream vs Downstream"
+    macro ../../../rradloff/FindCorrelation.C(\"mul\",\"yield_bcm_an_us\",\"yield_bcm_an_ds\",\"ErrorFlag == 0\",true,true)
```
Comment (leafybillow) [inline panguin/macros/bcm_calibrated.cfg]: @cameronc137 Please fix it.

**Comment by leafybillow** [comment]
@cipriangal It looks good to me. Thanks for taking care of this. 
@cameronc137 : For panguin macro commits, one cfg is calling FindCorrelation.C macro from Robert's own directory and it is not included in your commit. Please verify functionality before commit. 
Also, I would suggest replace the path to macros with an environment variable e.g. $QW_PATH. Panguin will get confused if it is called from other directory.

File: panguin/macros/bcm_calibrated.cfg [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# Defined cuts
+definecut evcut  (event_number>1)
+definecut bcmds_cut (-1*bcm_ds>1)
+
+# Uncomment the below line to monitor the fill of a ROOTfile.
+#watchfile
+
+newpage 3 3
+    title BCMs
+    macro ./macros/noStatBox.C()
```
Comment (leafybillow) [inline panguin/macros/bcm_calibrated.cfg]: Need a proper defined path to macro.

**Comment by cameronc137** [comment]
Ok, this looks great to me. These extra macro problems can be fixed on their own later. I put them into this pull request to allow for easier merging with operations later, and operations will have these fixes in due time.

**Pull request merged**